src/dialogs/config-flow/step-flow-pick-handler.ts

```diff
@@ -69,6 +74,28 @@ class StepFlowPickHandler extends LitElement {
         slug: handler,
       }));
 
+      // Get Supported Brands in a digestable list of objects
+      const supportedBrands: Array<SupportedBrandObj> = [];
+      for (const domain of Object.keys(h.supportedBrands)) {
+        const domainBrands = h.supportedBrands[domain];
+        for (const supportedBrand of Object.keys(domainBrands)) {
+          const index = supportedBrands.findIndex(
```

Comment:
We don’t support multiple so just override.

Reply:
We will support multiple eventually

ZHA, Zigbee for example

Reply:
Updated for now to replace